### PR TITLE
Add stdlib shim.

### PR DESCRIPTION
**Pull request by Lukasa**
This shims the stdlib ssl module into the PEP 543 interface.

### CODE REVIEW

File: pep543/stdlib.py [hunk 1]
```diff
@@ -0,0 +1,446 @@
...
+    """
+    if cert_chain:
+        # FIXME: support multiple certificates at different filesystem
+        # locations. This requires being prepared to create temporary
+        # files.
+        assert len(cert_chain[0]) == 1
+        cert_path = cert_chain[0][0]._cert_path
+        key_path = None
+        password = None
+        if cert_chain:
```
Comment (LilyFirefly) [inline pep543/stdlib.py]: Should this be `cert_path` or `cert_chain[1]`? `cert_chain` has already been checked.

Comment (Lukasa) [reply pep543/stdlib.py]: Should be `cert_chain[1]` I think, actually. Good spot.

File: test/test_stdlib.py [hunk 2]
```diff
@@ -2,10 +2,103 @@
...
+CONTEXTS = (
+    pep543.stdlib.STDLIB_BACKEND.client_context,
+    pep543.stdlib.STDLIB_BACKEND.server_context
+ )
+
+
+def wrap_buffers(context):
+    """
+    A convenient helper that calls wrap_buffers with the appropriate number of
+    arugments.
```
Comment (LilyFirefly) [inline test/test_stdlib.py]: ~~`arugments`~~ `arguments`

File: test/test_stdlib.py [hunk 3]
```diff
@@ -2,10 +2,103 @@
...
+        version raises a TLSError with Client contexts.
+        """
+        config = pep543.TLSConfiguration(
+            validate_certificates=False,
+            lowest_supported_version=lowest,
+            highest_supported_version=highest
+        )
+        ctx = context(config)
+        with pytest.raises(pep543.TLSError):
+            wrap_buffers(ctx)
```
Comment (LilyFirefly) [inline test/test_stdlib.py:57]: Are `None` and `object()` also bad values for the maximum version?

File: test/test_stdlib.py [hunk 4]
```diff
@@ -2,10 +2,103 @@
...
+
+    @pytest.mark.parametrize('context', CONTEXTS)
+    def test_no_supported_cipher_suites(self, context):
+        """
+        Using TLSConfiguration objects that have only unsupporetd cipher suites
```
Comment (LilyFirefly) [inline test/test_stdlib.py]: ~~`unsupporetd`~~ `unsupported`

File: test/test_stdlib.py (hunk 3 above)
Comment (Lukasa) [reply test/test_stdlib.py:57]: They are, and the test does validate this: only the docstring is wrong.

**Pull request merged**